extensions/intellij/intellij_generator_plugin/build.gradle

```diff
@@ -40,6 +40,7 @@ compileTestKotlin {
 patchPluginXml {
     changeNotes """
      <ul>
+        <li>v1.4.2 - Added an action item in the Project Menu to allow boostrapping BloC Tests based on the available/missing folders & files - inside lib & test</li>
```

Comment:
Considering we are introducing new changes, let's bump the version of the plugin to 1.5.0